int32 cmd_surge_starboard
int32 cmd_surge_port
int32 cmd_sway_bow
int32 cmd_sway_stern
int32 cmd_heave_bow
int32 cmd_heave_stern